docs/modules/ROOT/pages/getting-started/roles_persmissions.adoc

```diff
@@ -1,231 +1,366 @@
+
+=== Editing the access of a user
+. Select the three dots next to the user that you want to edit. 
+. Select *Edit access*. 
+. In *Assign roles*, change the drop-down to the new role that you want. 
+* To view the permissions for each role, select *Show list*. 
+. Select *Save changes*. 
+
+== Permissions by role
+
+The following table shows the actions that each type of collaborator can perform: 
+
+.Permissions for each type of collaborator
 
 |===
-|Permissions for |Action |Contributor |Maintainer |Owner
+|Action |Contributor |Maintainer |Admin
+
+|*Workspace:* 
 
-.4+|Workspace
-|View
-^|✔️
-^|✔️
-^|✔️
+View a workspace
+^.>|✔️
+^.>|✔️
+^.>|✔️
 
-|Create
+|Create a workspace
 |
 |
+^.^|✔️
+
+|Update a workspace
 |
-|Update
+^.^|✔️
+^.^|✔️
 
+|Delete a workspace
+|
 |
+^.^|✔️
+
+|*Application:*
+
+View an application
+^.>|✔️
+^.>|✔️
+^.>|✔️
+
+|Create an application
 |
 |
-|Delete
+^.^|✔️
+
+|Update an application
 |
+^.^|✔️
+^.^|✔️
 
+|Delete an application
 |
 |
-.4+|Applications
-|View
-^|✔️
+^.^|✔️
 
-^|✔️
-^|✔️
-|Create
+|*Component:*
+
+View a component
+^.>|✔️
+^.>|✔️
+^.>|✔️
+
+|Create a component
 |
-^|✔️
+^.^|✔️
+^.^|✔️
 
-^|✔️
-|Update
+|Update a component
 |
-^|✔️
-^|✔️
+^.^|✔️
+^.^|✔️
 
-|Delete
+|Delete a component
 |
 |
-^|✔️
-.4+|Component
+^.^|✔️
+
+|*Environment:*
+
+View an environment
 
-|View
-^|✔️
-^|✔️
-^|✔️
-|Create
+^.>|✔️
+^.>|✔️
+^.>|✔️
 
+|Create an environment
 |
-^|✔️
-^|✔️
-|Update
+^.^|✔️
+^.^|✔️
+
+|Update an environment
 |
+^.^|✔️
+^.^|✔️
 
-^|✔️
-^|✔️
+|Delete an environment
 |
+^.^|✔️
+^.^|✔️
+
+|*Pipeline run:*
+
+View pipeline run
+
+^.>|✔️
+^.>|✔️
+^.>|✔️
+
+|Create pipeline run
+^.^|✔️
+^.^|✔️
+^.^|✔️
+
+|Update pipeline run
 |
+^.^|✔️
+^.^|✔️
+
+|Delete pipeline run 
 |
+^.^|✔️
+^.^|✔️
+
+|*IntegrationTestScenario:*
 
-^|✔️
-.4+|Environment
-|View
-^|✔️
-^|✔️
+View IntegrationTestScenario
 
-^|✔️
-|Create
+^.>|✔️
+^.>|✔️
+^.>|✔️
+
+|Create IntegrationTestScenario
+|
+^.^|✔️
+^.^|✔️
+
+|Update IntegrationTestScenario
 |
+^.^|✔️
+^.^|✔️
+
+|Delete IntegrationTestScenario
 |
-^|✔️
+^.^|✔️
+^.^|✔️
 
-|Update
+|*Enterprise Contract:* 
+
+View Enterprise Contract
+^.>|✔️
+^.>|✔️
+^.>|✔️
+
+|Create Enterprise Contract
+|
 |
 |
-^|✔️
-|Delete
 
+|Update Enterprise Contract
 |
 |
-^|✔️
-.4+|IntegrationTestScenario
-|View
+^.^|✔️
 
-^|✔️
-^|✔️
-^|✔️
-|Create
+|Delete Enterprise Contract
 |
+|
+^.^|✔️
+
+|*Release strategy:*
+
+View Release strategy
+
+^.>|✔️
+^.>|✔️
+^.>|✔️
 
-^|✔️
-^|✔️
-|Update
+|Create Release strategy
 |
-^|✔️
+^.^|✔️
+^.^|✔️
 
-^|✔️
-|Delete
+|Update Release strategy
 |
-^|✔️
-^|✔️
+^.^|✔️
+^.^|✔️
+|Delete Release strategy
+|
+^.^|✔️
+^.^|✔️
+
+|*Release Admission Plan:*
+View Release Admission Plan
 
-.4+|EnterpriseContract
-|View
-^|✔️
-^|✔️
-^|✔️
+^.>|✔️
+^.>|✔️
+^.>|✔️
 
-|Create
+|Create Release Admission Plan
 |
+^.^|✔️
+^.^|✔️
+
+|Update Release Admission Plan
 |
-^|✔️
-|Update
+^.^|✔️
+^.^|✔️
 
+|Delete Release Admission Plan
 |
+^.^|✔️
+^.^|✔️
+
+|*User:*
```

Comment:
The user section is confusing too. Admins will be able to add *existing* users to the workspace or remove users who was previously added to the workspace. Every roles within the workspace will be able to see other users and their roles in the workspace. You can't create new users or delete users from RHTAP per se (so they now have access to RHTAP or lose access to RHTAP in general).

Reply:
Does this help clarify:

(User section): 
View a user 
Add a user to a workspace 
Delete a user from a workspace

Guessing we should remove "update a user" completely? Or do we want to get into how you can adjust a user's permissions to your workspace. Not sure if that is confusing here...

Reply:
Also would the same ^ apply to the user group section?

Reply:
> (User section):
View a user
Add a user to a workspace
Delete a user from a workspace

Yes, that looks better.